packages/frontend/src/app/layout.tsx

```diff
@@ -16,11 +17,13 @@ export default function RootLayout({
 }) {
   return (
     <html>
-      <body>
-        {children}
-        <BackToTop />
-        <Footer />
-      </body>
+      <StyledComponentsRegistry>
```

Comment:
how about putting `<StyledComponentsRegistry>` inside `<body>`? NextJS's document is doing like this

Reply:
`<body>` contains three components. If we move `<StyledComponentsRegistry>` inside `<body>`, we needs to add one more `<div>` in the `<StyledComponentsRegistry>`. Therefore, to avoid adding more `<div>`, I add `<StyledComponentsRegistry>` outside the `<body>`.

Reply:
oh I see, thanks for explanation, LGTM!